kuryr_kubernetes/handlers/retry.py

```diff
@@ -83,6 +83,13 @@ def __call__(self, event, *args, **kwargs):
                 }
                 self._handler(event, *args, retry_info=info, **kwargs)
                 break
+            except (exceptions.LoadBalancerNotReady,
+                    exceptions.PortNotReady):
+                with excutils.save_and_reraise_exception() as ex:
+                    if self._sleep(deadline, attempt, ex.value):
+                        ex.reraise = False
+                    else:
```

Comment:
I'm fairly sure we don't need else-raise construct - the `excutils.save_and_reraise_exception()` context manager should do that for us.

Reply:
True I wasn't actually paying too much attention to if-else structure. its not needed here indeed.